packages/web-frontend/config/webpack.config.dev.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
       // Support React Native Web
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
       'react-native': 'react-native-web',
+      '@material-ui/styles': path.resolve(__dirname, '..', 'node_modules', '@material-ui/styles'),
```

Comment:
This prevents the package being loaded more than once

Reply:
Nice, add as a comment